spec/tasks/migration_rake_spec.rb

```diff
@@ -30,6 +31,8 @@
     end
 
     after(:each) do
+      @file.delete
+      @collection.delete
```

Comment:
cleanup_jetty call below removes all files and collections (and its somehow faster then individually removing them)